source/arriving/setup_computer.rst

```diff
@@ -1,35 +1,274 @@
 .. _ref_setupcomputer:
 
+.. role:: bash(code)
+   :language: bash
+
 Setting up your computer
 ========================
 
 Before you can start working with the tools commonly-used or developped in the lab, you need to setup your computer. Follow this step-by-step guide to get your computer ready to work.
 
 *This section is a work in progress. Eventually, it should cover OS choices, IDE choice, python and virtualenv installation, scilpy installation, nextflow and singularity/docker.*
 
-We recommend using linux, but Mac can be an option. With Windows, your life will definitely be more complicated.
+We recommend using Linux, but Mac and Windows can also be viable. If you already have a computer with Linux (or Mac) installed, proceed to the next step. If you want to install Linux on your personal computer but also want to keep Windows, you have two choices:
+
+    1) You can dual-boot Linux and Windows by following one of the many tutorials online, such as this `one <https://www.freecodecamp.org/news/how-to-dual-boot-windows-10-and-ubuntu-linux-dual-booting-tutorial/>`_. This process can be tedious so don't hesitate to ask for help in the lab.
+
+    2) You can use WSL2 on Windows 10 or 11. A step-by-step guide is available here (*TODO add link to new section*) to help you install WSL2.
+
+IDE
+"""
+
+An integrated development environment (IDE) is an application made for software development, including code editing, debugging, and testing. Most people at the lab use `VSCode <https://code.visualstudio.com/download>`_, which is always free and compatible on all platforms. Installing this first will be helpful for some of the next steps.
+
+Terminal and bash
+"""""""""""""""""
+
+Now is the time to open a bash terminal, also known as the command line. The terminal allows you to execute commands, manage files and folders, and automate tasks. You will use it to install all you need to complete the setup of your computer.
+
+It is useful to learn some of the basics of the bash language to help you play around. Please refer to this cheat sheet on the bash language (*TODO add link to new section*).
+
+Before proceeding to the next steps, here are some useful additions you can make to your terminal "settings", by modifying what we call the *.bashrc*. First, open the .bashrc in VSCode by typing in the terminal :bash:`code .bashrc` (make sure you are in your home directory by simply typing :bash:`cd`). Then, choose the features you like!
+
+* | If you want your terminal to display helpful colors and show your current git branch:
+
+.. code-block:: bash
+
+    # Replace this part:
```

Comment:
If people have a ubuntu installed by themselves, I'm not sure this part is always there. I think the default .bashrc is pretty much blank. It's the tecs here that fill it with so much stuff.